pxr/base/tf/stringUtils.cpp

```diff
@@ -63,6 +63,37 @@ using std::vector;
 
 PXR_NAMESPACE_OPEN_SCOPE
 
+// environment setting for defaulting back to standard ASCII rules for 
+// identifiers and prim names.  By default this value is true indicating 
+// that Unicode processing paths will be used on UTF-8 character sequences 
+// to determine identifier validity.  In addition, if this value is true, 
+// slightly different rules will be used to evaluate generic identifiers 
+// vs. prim names.  Setting this value to false will fallback to original
+// ASCII identifier rules.
+TF_DEFINE_ENV_SETTING(TF_UTF8_IDENTIFIERS, 
+    true, 
+    "Allow UTF8 strings as identifiers and prim names");
+
+namespace {
+    bool _IsASCIIValue(const char& c)
```

Comment:
The C++ core guidelines recommend cheap to copy types be passed by value instead of by reference.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in